Documentation/How to Deploy an IoT App.md

```diff
@@ -0,0 +1,91 @@
+# How to Deploy an IoT App
+
+This provides information how to prepare a Publish Profile and deploy an application to a development board.
+
+## How to Prepare a Publish Profile
```

Comment:
The doc looks great overall, the only thing I would remove is this section given that Publish profile is really only applicable to the VS scenario. When you do it through the command line, you are not really creating a publish profile, which is kind of why I think that just removing this section and sentence would be better, And just having this doc have basically 4 sections: Using Visual Studio, using .net cli, copying to target device and references.

Reply:
Gotcha.  One side note... The **win-arm** and **linux-arm** runtime-identifiers may become default at some point.   See https://github.com/aspnet/websdk/issues/455 for details.